react-context/client/src/pages/HomePage.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars -- Remove me */
+/* eslint-disable no-undef -- Remove me */
+import { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import AppContext from '../components/AppContext';
+import './HomePage.css';
+
+export default function Home() {
+  /* TODO: Grab `user` from `AppContext` */
+  const navigate = useNavigate();
+  const user = useContext(AppContext);
```

Comment:
This should be destructured to get the `user`. As it is, this is the entire object, so you may accidentally think they are logged in when they actually are not.